src/twisted/internet/test/test_inlinecb.py

```diff
@@ -83,6 +96,26 @@ def _genBasics():
 
         return _genBasics().addCallback(self.assertEqual, "WOOSH")
 
+    def testBasicsAsync(self):
+        """
+        Test that a inlineCallbacks that yields coroutine works.
+        Alse tests functions that throw exceptions.
```

Comment:
This suggetion started due to the typo...  and I ended up with this. Based on https://jml.io/test-docstrings/ :)

```suggestion
        C{inlineCallbacks} can yield a coroutine and catch its exception.
```

Reply:
Thanks, this is way clearer.